### PR TITLE
chore: code formatting, tidy up errors, remove unused functions

### PR DESCRIPTION
**Pull request by shreddedbacon**
# Checklist
- [ ] Affected Issues have been mentioned in the Closing issues section
- [ ] Documentation has been written/updated.
- [ ] Changelog entry has been written

Just fixes some minor code formatting issues, tidies up some errors and also removes some unused functions and variables.

### CODE REVIEW

File: pkg/lagoon/users/main.go [hunk 1]
```diff
@@ -106,7 +106,7 @@ func (ud *Data) AddItem(userData UserData) {
 func distinctObjects(objs []UserData) (distinctedObjs []UserData) {
```
Comment (bomoko) [inline pkg/lagoon/users/main.go:106]: I think that given we're cleaning stuff up, this function (and the next) can probably be made tighter with the new slices library.

Maybe something like
```
func distinctObjects(objs []UserData) (distinctedObjs []UserData) {
	var output []UserData

	ids := []string{}

	for _, e := range objs {
		if !slices.Contains(ids, e.ID) {
			ids = append(ids, e.ID)
			output = append(output, e)
		}
	}

	return output
}
```

**Review by bomoko — APPROVED** [review]
Looks good! Just suggested one thing to thing about.

File: pkg/lagoon/users/main.go (hunk 1 above)
Comment (shreddedbacon) [reply pkg/lagoon/users/main.go:106]: Ahh nice. I didn't go too deep into a lot of the code itself more formatting. But this seems simple enough to slide in

Comment (shreddedbacon) [reply pkg/lagoon/users/main.go:108]: Updated

**Pull request merged**